### DISCUSSION TITLE
SSH Tunnel fail when running VPN

### DISCUSSION THREAD

**Discussion by I33Buckler** [dbeaver/dbeaver#11292] [Q&A]
#### System information: 
- Debian 5.9.15-1~bpo10+1 (2020-12-31) x86_64 GNU/Linux
- DBeaver version 7.3.4.202101311042

#### Connection specification:
- Postgresql 11 
- SSH tunneling
- 
#### Describe the problem you're observing:
When running NordVPN I am unable to connect to the Postgresql with DBeaver.

#### Steps to reproduce, if exist:
Note the following when connecting directly to the database from the terminal.
1) Start NordVPN: nordvpn c
2) Connect SSH tunnel: ssh -L 2222:<remote-host>:5432 <user>@<remote-host>
    Connection is successful.
3) Connect to database via SSH tunnel: psql -h localhost -d <database> -p 6666 -U <user>

when using DBeaver there is an issue with connection when running NordVPN.
The connection is successful when the VPN is not running.
However, 
when running the VPN DBeaver reports the following.
Can't initialize tunnel
  Cannot establish tunnel
  Cannot establish tunnel
    java.net.UnknownHostException: canberramjg.ddns.net
    java.net.UnknownHostException: canberramjg.ddns.net
      canberramjg.ddns.net
      canberramjg.ddns.net
Attached is the DBeaver logs
[DBeaver with VPN.log](https://github.com/dbeaver/dbeaver/files/5938203/DBeaver.with.VPN.log)

**Comment by mwalkerr** [comment]
@I33Buckler Are you using SSHJ? There was some other issue I had in the past where using that instead of "JSch" worked for me. See https://github.com/dbeaver/dbeaver/issues/5845#issuecomment-541593022
